### PR TITLE
feat(actions): Added delete eph action and reworked data exchange between steps

### PR DESCRIPTION
**Pull request by Pive01**
Added action to remove eph instance once done with testing and reworked how data is exchanged switching from upload/download of artifacts to use of env vars

### CODE REVIEW

File: .github/workflows/preview.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  preview-test:
+    name: 'Test preview workflow'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Deploy Preview
+        uses: ./preview
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (lukqw) [inline .github/workflows/preview.yml:14]: maybe @HarshCasper knows why the token can't comment on the PR?

File: remove-eph-instance/action.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+name: Delete Preview Environment
+
+inputs:
+  localstack-api-key:
+    description: 'LocalStack API key used to access the preview environment'
+    required: true
+  pr_id:
```
Comment (lukqw) [inline remove-eph-instance/action.yml:7]: should be preview_id

File: preview/action.yml [hunk 3]
```diff
@@ -50,15 +53,9 @@ runs:
...
-        echo $endpointUrl > ./ls-preview-url.txt
         echo "LS_PREVIEW_URL=$endpointUrl" >> $GITHUB_ENV
         echo "AWS_ENDPOINT_URL=$endpointUrl" >> $GITHUB_ENV
-
-    - name: Upload preview instance URL
-      uses: actions/upload-artifact@v3
-      with:
-        name: pr
-        path: ./ls-preview-url.txt
+        echo "LS_PREVIEW_URL=$endpointUrl" >> $GITHUB_OUTPUT
```
Comment (lukqw) [inline preview/action.yml:58]: missing in outputs

**Review by lukqw — COMMENTED** [review]
As discussed in sync, please reach out to @HarshCasper to discuss whether we want to stick with artifacts based value passing, or if we should switch to env var OR input/output based method.

To me input/output based seems most robust

File: preview/action.yml (hunk 3 above)
Comment (Pive01) [reply preview/action.yml:58]: It is actually defined at line 19

**Comment by Pive01** [comment]
Also this PR has potentially breaking changes so might be a good idea to have a new release after merging it and maybe tag it with v1? Usually I've seen mostly versioning of GH actions with v1 v2 v3 ecc... not much v0.1.3

**Comment by lukqw** [comment]
I'm superseding this with #19 as I'd like to break this up into two steps:
1) introduce the shutdown action
2) refactor artifacts data exchange into inputs/outputs (or env var based) exchange with feedback from @lakkeger.

**Pull request closed without merging**